azure-pipelines.yml

```diff
@@ -109,9 +109,28 @@ jobs:
       displayName: Install Qt5 and pkg-config prereqs
     - script: sh -x ./clean_build.sh
       displayName: Clean Build
+
+- job: macOSRelease
```

Comment:
I think it would be better to merge this to bug-sur-Release and add a condition on the artifact publishing task so that it only happens on big-sur job.

Reply:
Sorry, could you explain further? I've made some changes locally that now build on all three Mac build agents both Debug and Release mode.

Reply:
There is *one* issue with Universal DMGs: basically, Qt 5 doesn't support ARM64 - only Qt 6, it would appear. So we would have to look into bumping to Qt 6, either on Mac only or globally. It's not ideal. So for now, I think we should pause the Universal DMG directive in CMakeLists.txt, and explore bumping to Qt 6 in a separate issue. Thoughts?

Reply:
Resolving, as this review comment has been addressed.